requirements.txt

```diff
@@ -0,0 +1,6 @@
+canonicalwebteam.flask-base==0.1.0
+canonicalwebteam.templatefinder==0.1.1
+Flask==1.0.2
+python-frontmatter==0.4.5
```

Comment:
@nottrobin Are this not installed by `canonicalwebteam.templatefinder==0.1.1`

Reply:
Yeah, @sowasred2012 please without this dependency

Reply:
Same for `mistune` and `bleach`.